packages/icons/icons.json

```diff
@@ -4590,5 +4590,12 @@
         "tags": "shape, 3d, object, box, container, view",
         "group": "miscellaneous",
         "codepoint": 62352
+    },
+    {
+        "displayName": "Source",
+        "iconName": "source",
```

Comment:
The name "source" feels very generic / doesn't really help describe what this icon is. Can we consider renaming this to be something more descriptive? Something like: `database-cloud`